projects/admin/src/app/circulation/item/item.component.html

```diff
@@ -58,19 +58,10 @@
     <a [routerLink]="['/records','documents','detail', item.document.pid]" *ngIf="item.document.title | mainTitle as title ">
       {{ isCollapsed ? (title | truncateText: 12) : title }}
     </a>
-    <ng-container *ngIf="document && document.contribution && document.contribution.length > 0">
-      <ul class="list-inline mb-0 small">
-        <li *ngFor="let contribution of document.contribution; let last = last; let i = index" class="list-inline-item">
-          <span *ngIf="!contribution.agent.pid" [innerHTML]="contribution | contributionFormat: true"></span>
-          <a
-            *ngIf="contribution.agent.pid"
-            [innerHTML]="contribution | contributionFormat: true"
-            [routerLink]="['/records', 'persons', 'detail', contribution.agent.pid]">
-          </a>
-          {{ last ? '' : '; ' }}
-        </li>
-      </ul>
-    </ng-container>
+    <shared-contribution
+      *ngIf="document?.contribution && document.contribution.length > 0"
```

Comment:
COSMETIC : `*ngIf` seems not necessary. We could use the notation `[contributions]="document?.contribution` instead. This could be this component that check if input is empty or not